i2c/oled_i2c/img_to_array.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+# Converts a greyscale image into a format able to be
+# displayed by the SSD1306 driver in horizontal addressing mode
+
+# usage: ./img_to_array.py <logo.bmp> on a Raspberry Pi
+# or python3 img_to_array.py <logo.bmp>
+
+# depends on the Pillow library
+# `python3 -m pip install --upgrade Pillow`
+
+from PIL import Image
+import sys
+
+OLED_HEIGHT = 32
+OLED_WIDTH = 128
+OLED_PAGE_HEIGHT = 8
+OLED_NUM_PAGES = OLED_HEIGHT / OLED_PAGE_HEIGHT
```

Comment:
This isn't used anywhere?

Reply:
We use `OLED_NUM_PAGES` and I didn't want that to be a magic number so I added the rest of the variables in. Could also be useful if someone decides to use the 128x64 version?

Reply:
I agree that the rest of them ought to be kept, but it seems like `OLED_NUM_PAGES` isn't actually used in the Python script? (which is actually what I meant with my previous comment, apologies for the ambiguity)

Reply:
Oh true!